### PR TITLE
Fix 5326 Improve add group action to support override of options

### PR DESCRIPTION
**Pull request by allyoucanmap**
## Description

This PR adds a new parameter to `addGroup` action called `options`.
The `options` parameter will override generated option of group.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Other... Please describe: Improvement of existing action

## Issue

**What is the current behavior?**

#5326

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [x] No

### CODE REVIEW

File: web/client/actions/__tests__/layers-test.js [hunk 1]
```diff
@@ -331,4 +331,17 @@ describe('Test correctness of the layers actions', () => {
...
+        const options = {
+            id: 'uuid',
+            title: 'My title',
+            expanded: true
+        };
+        const action = addGroup(options.id, undefined, options);
+        expect(action.type).toBe(ADD_GROUP);
+        expect(action.group).toBe(options.id);
+        expect(action.parent).toNotExist();
+        expect(action.options).toBe(options);
```
[the remark this answers is not in the document]
Comment (allyoucanmap) [reply web/client/actions/__tests__/layers-test.js:345]: ok, I will change `toBe` in `toEqual` and also `toNotExist` with `toBeFalsy`.

File: web/client/actions/layers.js [hunk 2]
```diff
@@ -183,12 +183,20 @@ function addLayer(layer, foreground = true) {
...
 }
-
-function addGroup(group, parent) {
+/**
+ * Add a new group
+ * @memberof actions.layers
+ * @function
+ * @param {string} group title of group
+ * @param {string} parent parent group described with dot notation (eg parent.nested )
+ * @param {object} options override all option of a group while adding it
```
[the remark this answers is not in the document]
Comment (allyoucanmap) [reply web/client/actions/layers.js:192]: @mbarto @offtherailz @MV88 If everyone agree I'll change as you suggested, I have not preference on this.

[the remark this answers is not in the document]
Comment (mbarto) [reply web/client/actions/layers.js:192]: @allyoucanmap I prefer your original version

[the remark this answers is not in the document]
Comment (offtherailz) [reply web/client/actions/layers.js:192]: I agree with @mbarto. Additionally, I should not specify "overide all options".  
```suggestion
 * @param {object} options Additional properties to assign to the group. They will override the default ones.
```

**Pull request merged**